### PR TITLE
[Dungeon] Sunken Temple

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.
Missing AI included now as well.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.

Make sure to add very very basic gear towards each Dungeon.

### CODE REVIEW

**Comment by merdlin** [comment]
tested and it works.

**Comment by BarbzYHOOL** [comment]
how did you test @merdlin ?

**Comment by merdlin** [comment]
just like every test I do, I ran the modified files and got equipped in the game and went to attack the bosses

**Comment by Knindzagxg** [comment]
Make sure to check there are no issues with boss damage as well.
Since we using new Values, their damage is increased and if i made some mistake that get you killed in 1-2 shoots, post here pretty please.

File: data/sql/updates/pending_db_world/rev_1588338380679685500.sql [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 344, `maxdmg` = 444, `DamageModifier` = 1.01 WHERE `entry` = 5713;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 344, `maxdmg` = 444, `DamageModifier` = 1.01 WHERE `entry` = 5714;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 344, `maxdmg` = 444, `DamageModifier` = 1.01 WHERE `entry` = 5715;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 344, `maxdmg` = 444, `DamageModifier` = 1.01 WHERE `entry` = 5716;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 344, `maxdmg` = 444, `DamageModifier` = 1.01 WHERE `entry` = 5717;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 310, `maxdmg` = 400, `DamageModifier` = 1.01 WHERE `entry` = 5719;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 300, `maxdmg` = 388, `DamageModifier` = 1.01 WHERE `entry` = 5720;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 310, `maxdmg` = 400, `DamageModifier` = 1.01 WHERE `entry` = 5721;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 300, `maxdmg` = 488, `DamageModifier` = 1.01 WHERE `entry` = 5722;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588338380679685500.sql:56]:
```suggestion
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 410, `maxdmg` = 500, `DamageModifier` = 1.01 WHERE `entry` = 8580;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 500, `maxdmg` = 788, `DamageModifier` = 1.01 WHERE `entry` = 8443;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 716, `maxdmg` = 908, `DamageModifier` = 1.01 WHERE `entry` = 5709;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 328, `maxdmg` = 421, `DamageModifier` = 1.01 WHERE `entry` = 5710;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 344, `maxdmg` = 444, `DamageModifier` = 1.01 WHERE `entry` IN (5711, 5717, 5712, 5713, 5714, 5715, 5716);
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 310, `maxdmg` = 400, `DamageModifier` = 1.01 WHERE `entry` IN (5719, 5721);
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 300, `maxdmg` = 388, `DamageModifier` = 1.01 WHERE `entry` IN (5720, 5722);
```

File: data/sql/updates/pending_db_world/rev_1588338380679685500.sql [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+UPDATE `creature_template` SET `mindmg` = 180, `maxdmg` = 244, `DamageModifier` = 1.03 WHERE `entry` = 5280;
+UPDATE `creature_template` SET `mindmg` = 383, `maxdmg` = 520, `DamageModifier` = 1.03 WHERE `entry` = 8317;
+UPDATE `creature_template` SET `mindmg` = 180, `maxdmg` = 244, `DamageModifier` = 1.03 WHERE `entry` = 5271;
+UPDATE `creature_template` SET `mindmg` = 180, `maxdmg` = 244, `DamageModifier` = 1.03 WHERE `entry` = 5273;
+UPDATE `creature_template` SET `mindmg` = 172, `maxdmg` = 234, `DamageModifier` = 1.03 WHERE `entry` = 5269;
+UPDATE `creature_template` SET `mindmg` = 187, `maxdmg` = 253, `DamageModifier` = 1.03 WHERE `entry` = 5263;
+UPDATE `creature_template` SET `mindmg` = 180, `maxdmg` = 244, `DamageModifier` = 1.03 WHERE `entry` = 5291;
+UPDATE `creature_template` SET `mindmg` = 116, `maxdmg` = 154, `DamageModifier` = 1.03 WHERE `entry` = 8497;
+UPDATE `creature_template` SET `mindmg` = 59, `maxdmg` = 80, `DamageModifier` = 1.03 WHERE `entry` = 8437;
+UPDATE `creature_template` SET `mindmg` = 363, `maxdmg` = 480, `DamageModifier` = 1.03 WHERE `entry` = 8438;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588338380679685500.sql:36]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 191, `maxdmg` = 259, `DamageModifier` = 1.03 WHERE `entry` IN (5226, 8384, 5228, 5267);
UPDATE `creature_template` SET `mindmg` = 182, `maxdmg` = 248, `DamageModifier` = 1.03 WHERE `entry` = 8311;
UPDATE `creature_template` SET `mindmg` = 177, `maxdmg` = 239, `DamageModifier` = 1.03 WHERE `entry` IN (5259, 5259);
UPDATE `creature_template` SET `mindmg` = 187, `maxdmg` = 253, `DamageModifier` = 1.03 WHERE `entry` IN (8318, 5263);
UPDATE `creature_template` SET `mindmg` = 57, `maxdmg` = 76, `DamageModifier` = 1.03 WHERE `entry` IN (8656, 8657);
UPDATE `creature_template` SET `mindmg` = 44, `maxdmg` = 61, `DamageModifier` = 1.03 WHERE `entry` = 8658;
UPDATE `creature_template` SET `mindmg` = 63, `maxdmg` = 85, `DamageModifier` = 1.03 WHERE `entry` = 8324;
UPDATE `creature_template` SET `mindmg` = 195, `maxdmg` = 259, `DamageModifier` = 1.03 WHERE `entry` IN (5228, 5256, 5228);
UPDATE `creature_template` SET `mindmg` = 195, `maxdmg` = 264, `DamageModifier` = 1.03 WHERE `entry` = 5283;
UPDATE `creature_template` SET `mindmg` = 199, `maxdmg` = 264, `DamageModifier` = 1.03 WHERE `entry` = 5277;
UPDATE `creature_template` SET `mindmg` = 180, `maxdmg` = 244, `DamageModifier` = 1.03 WHERE `entry` IN (5280, 5271, 5273, 5291);
UPDATE `creature_template` SET `mindmg` = 383, `maxdmg` = 520, `DamageModifier` = 1.03 WHERE `entry` = 8317;
UPDATE `creature_template` SET `mindmg` = 172, `maxdmg` = 234, `DamageModifier` = 1.03 WHERE `entry` = 5269;
UPDATE `creature_template` SET `mindmg` = 116, `maxdmg` = 154, `DamageModifier` = 1.03 WHERE `entry` = 8497;
UPDATE `creature_template` SET `mindmg` = 59, `maxdmg` = 80, `DamageModifier` = 1.03 WHERE `entry` = 8437;
UPDATE `creature_template` SET `mindmg` = 363, `maxdmg` = 480, `DamageModifier` = 1.03 WHERE `entry` = 8438;
```

**Comment by Kitzunu** [comment]
Ty for the PR!

**Pull request merged**